SalesforceMobileSDK-iOS.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
       smartstore.dependency 'SQLCipher', '~> 3.1'
       smartstore.dependency 'SQLCipher/fts', '~> 3.1'
-      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
+      smartstore.dependency 'FMDB', '~> 2.5'
+      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
```

Comment:
We lost it only three hours ago: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/commit/9263978ade2ffb6990635713e4f97e3dd28f6844

I'll put it back